src/main/java/org/usfirst/frc/team3786/robot/subsystems/GrabberSubsystem.java

```diff
@@ -23,10 +23,10 @@ public static GrabberSubsystem getInstance() {
 	}
 
 	public GrabberSubsystem() {
-		grabber = new WPI_TalonSRX(Mappings.grabberMotor);
+		//grabber = new WPI_TalonSRX(Mappings.grabberMotor);
```

Comment:
I think you still need these (grabber and flinger).

Reply:
What are the IDs for the grabbers going to be? It looks like I replaced the grabber and flinger motors with the buttlifter and roller motor IDs.

Reply:
You can define new values for the CAN IDs for grabber and flinger in Mappings.java.

Reply:
Can I use any value, like 8 for the grabber and 9 for the flinger?

Reply:
Yes, that's fine.